cmd/admin-heal.go

```diff
@@ -27,7 +28,17 @@ import (
 	"github.com/minio/minio/pkg/madmin"
 )
 
+const (
+	scanNormalMode = "normal"
+	scanDeepMode   = "deep"
+)
+
 var adminHealFlags = []cli.Flag{
+	cli.StringFlag{
+		Name:  "scan",
+		Usage: "select the healing scan mode",
+		Value: scanNormalMode,
```

Comment:
@vadmeste can you provide a way for the user to know how deep scan differs from the normal scan?

Reply:
> @vadmeste can you provide a way for the user to know how deep scan differs from the normal scan?

I added more text in the HELP of admin heal command